test/plugins/features/movies/index.test.js

```diff
@@ -20,4 +22,33 @@ describe('movies integration', () => {
 
   });
 
+  describe('fetch', () => {
+
+    it('fetches a movie', () => {
+      return Movies.inject({
+        url: '/movies/1',
+        method: 'GET'
+      })
+      .then((response) => {
+        expect(response.statusCode).to.eql(200);
+      });
+
+    });
+
+  });
+
+  describe('fetchAll', () => {
+
+    it('fetches all movies', () => {
+      return Movies.inject({
+        url: '/movies',
+        method: 'GET'
+      })
+      .then((response) => {
+        expect(response.statusCode).to.eql(200);
```

Comment:
Can you assert the length of the movies list here? Or at least check the length isn't 0.

Reply:
added new assertion that response.result to not be empty